soa/service/remote_credential_provider.cc

```diff
@@ -1,188 +0,0 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
```

Comment:
Consider keeping this.  It provides an escape hatch for if the user doesn't want to trust internal credentials, and is orthogonal to the rest of the changes.